index.ts

```diff
@@ -33,6 +33,7 @@ export interface DocWithErrors extends DocBase {
 export interface DocBase {
 	jsonapi?: ImplementationInfo;
 	links?: Links | PaginationLinks;
+	meta?: MetaObject;
```

Comment:
Can you write some tests for this, following examples like this
https://github.com/mike-north/jsonapi-typescript/blob/master/tests/resources/main.test.ts

Reply:
Will do.

Nice, I'm even learning how to test TS! 😃